src/resources/report.js

```diff
@@ -20,12 +19,33 @@ export default api => (
       updated_at: T.object,
     }
 
+    constructor(data = {}) {
+      super(data);
+      if (data.type) this._url = this.constructor.constructUrl(data.type);
```

Comment:
linter isn't catching it. looked for that eslint setting in their docs, but didn't find it.